docs/source/install.rst

```diff
@@ -22,71 +22,47 @@ The following versions of Python can be used: 2.7.6+, 3.4.3+, 3.5.1+, and 3.6.0+
 
 .. note::
 
-   We are testing CuPy automatically with Jenkins, where all the above *recommended* environments are tested.
+   We are testing CuPy automatically with Jenkins, where all the *recommended* environments above are tested.
    We cannot guarantee that CuPy works on other environments including Windows and macOS, even if CuPy looks running correctly.
 
-CuPy uses C++ compiler such as g++.
-You need to install it before installing CuPy.
-This is typical installation method for each platform::
-
-  # Ubuntu 14.04
-  $ apt-get install g++
-
-  # CentOS 7
-  $ yum install gcc-c++
-
-If you use old ``setuptools``, upgrade it::
+Before installing CuPy, we recommend you to upgrade ``setuptools`` and ``pip``::
```

Comment:
* On Ubuntu 14.04, users cannot install wheels without the latest ``pip``.
* ``g++`` is not required for wheel users, so I moved it to the "Install CuPy from Source" section.